doc/overview.rst

```diff
@@ -2394,15 +2448,15 @@ configuration file.
 -  Model: ``CreateModel``, ``UpdateModel``
 -  Model Package: ``CreateModelPackage``, ``UpdateModelPackage``
 -  Processing Jobs: ``CreateProcessingJob``, ``UpdateProcessingJob``
--  Training Job: ``CreateTrainingJob``, ``UpdateTrainingJob``
--  Edge Packaging Job: ``CreateEdgePackagingJob``, ``UpdateEdgePackagingJob``
+-  Training Job: ``CreateTrainingJob``
+-  Edge Packaging Job: ``CreateEdgePackagingJob``
 
 Hyperparameter Tuning Job: Supported indirectly via ``TrainingJob`` API. While this API is not directly supported, it includes the training job definition as a parameter.
 If you provide defaults for this parameter as part of the ``TrainingJob`` API, these defaults are also used for Hyperparameter Tuning Job.
 
-The following goups of SDK capabilities support defaults with a configuration file.
+The following groups of SDK capabilities support defaults with a configuration file.
```

Comment:
Under here we should mention the new SDK specific parameters as well (for Session and Estimator), right?